README.md

```diff
@@ -50,7 +50,11 @@ Finally, follow the instructions displayed by the virtual machine.
 
 ### Run without Vagrant
 
-You can also run this project locally without using Vagrant. First [install Foreman globally](https://github.com/ddollar/foreman#installation):
+You can also run this project locally without using Vagrant, but you will need
+to make sure the "required packages" from `script/provision.sh` are available
+locally, before you begin.
+
```

Comment:
:+1: for this addition. It would probably be useful to also add what these are likely to be on something other than debian (e.g. brew), but that can be for later!